### PR TITLE
Clean up.

### PR DESCRIPTION
**Pull request by recena**
@reviewbybees, specially @rsandell and @jtnord

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.580.1</version>
+        <version>1.580.3</version>
```
Comment (oleg-nenashev) [inline pom.xml]: :bug: It forces the plugin to be incompatible with user installations on 1.580.1

**Comment by oleg-nenashev** [comment]
:bug: unjustified core dependency update, which impacts users on 1.580.1

File: pom.xml (hunk 1 above)
Comment (recena) [inline pom.xml]: I thought the idea was work with the latest release of a specific LTS version.

Comment (recena) [inline pom.xml]: Reverted.

**Comment by oleg-nenashev** [comment]
:bee:

**Comment by kzantow** [comment]
:bee:

**Comment by recena** [comment]
@reviewbybees done

**Comment by stephenc** [comment]
:bee:

**Pull request merged**